uwblueprint/childrens-aid-society
# Create Caregivers Model

## Notion ticket link

[Create Caregivers Model](https://www.notion.so/uwblueprintexecs/Create-caregivers-model-133350244f2c424bbef30493a6981310)



## Implementation description
* Create the caregivers model in a new models/caregivers.py file so it matches the schema 
* Generated the migration and checked if working



## Steps to test
1. Get the database using: docker exec -it cas_db psql -U postgres -d cas
2. select * from caregivers;
3. You should now see the table with the values



## What should reviewers focus on?
* On files: caregivers.py in python/app/models
* in migrations/versions/...create_caregivers_models


## Checklist
